[Web] Create RoomItem molecule

moaikang opened the pull request:
### 🍞 Issue Number 

close: #174 
<br/>

### 🥞 작업 내역

<img width="514" alt="스크린샷 2020-12-05 오후 5 32 43" src="https://user-images.githubusercontent.com/52201658/101237936-fe9abe00-371f-11eb-979d-e143b40a9996.png">
<img width="322" alt="스크린샷 2020-12-05 오후 5 32 53" src="https://user-images.githubusercontent.com/52201658/101237938-00648180-3720-11eb-929e-2b9865d6e4bc.png">

> 구현 내용 및 작업 했던 내역

- [ ] RoomItem molecule 제작
- [ ] RoomItem Story 제작
<br/>


### 🍞 새로운 기능

- RoomItem molecule
  <br/>

### 🍞 작업 유형

- [x] 신규 기능 추가
- [ ] 버그 수정
- [ ] 리펙토링
- [ ] 문서 업데이트
  <br/>

### 🥞 체크리스트

- [x] Merge 하는 브랜치가 올바른가?
- [x] 코딩컨벤션을 준수하는가?
- [x] PR과 관련없는 변경사항이 없는가?
- [x] 내 코드에 대한 자기 검토가 되었는가?
- [ ] 변경사항이 효과적이거나 동작이 작동한다는 것을 보증하는 테스트를 추가하였는가?
- [ ] 새로운 테스트와 기존의 테스트가 변경사항에 대해 만족하는가?
  <br/>

### 🥞 PR 특이 사항

> PR을 볼 때 주의깊게 봐야하거나 말하고 싶은 점

- Text 컴포넌트에 className 속성을 넘겼음.
- 이는 styled(MyComponent) 함수가 className으로 스타일을 주입하기 때문임.
[참고 링크](https://styled-components.com/docs/basics#styling-any-component)

<br/><br/>

Commit:
[Web] Feat : :zap:️ create RoomItemBox's story #174

RoomItemBox의 story를 만들었다.

Commit:
[Web] Feat : :zap:️ create RoomItemBox molecule #174

RoomItemBox molecule을 만들었다. RoomItemBox에 필요한 padding을 추가했고,
Text를 styled(Text) 문법을 사용하기 위해 props로 className을 추가하였다.

pje1740 commented on client/src/components/molecules/common/RoomItem.tsx:
```diff
@@ -0,0 +1,56 @@
+              {createdAt}
+            </MarginedText>
+          </RoomItemHeaderLeft>
+          <Text size={14} color={Palette.DARK_GREY}>{`(${participantCount} / 8)`}</Text>
```
8로 적어도 사실 기획상 8명 고정이라 상관 없을 거 같지만 이것도 확장성을 고려한다면 props로 받아오는 게 좋으려나요?? 우선 지금은 급하게 수정될 부분은 아닌 거 같은데 보면서 든 생각이라 적어봐요! ㅎㅎㅎ

moaikang replied:
이거 저도 엄청 고민했습니다! 이거 지금 빼놓을까 생각했는데 우선 이렇게 하고 리팩터할때 빼려구요..!

pje1740 commented on client/src/stories/molecules/RoomItemType.stories.tsx:
```diff
@@ -0,0 +1,27 @@
+  size: 'small',
+  createdAt: '몇 분 전 생성',
+  participantCount: 3,
+  title: '에반데',
```
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ예쁜말로 수정 가능할까욬ㅋㅋㅋㅋㅋㅋㅋ

kas136 replied:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

pje1740 reviewed (commented):
Molecule 합치는 거 진짜 은근 손이 많이 갈 거 같긴하네요...! 보면서 참고해봐야겠습니다!! 고생하셨어요!!

Commit:
[Web] Fix : :fire: fix story content and add roomCapacity type #174

story의 문구를 수정하고, 추후 최대 방 인원이 늘어났을 때를 대비하여 최대 방 인원을 props로 넘겨서 렌더링 하는 방법으로 바꾸었다.

moaikang commented:
수정 완료했습니다 🌈🌈

pje1740 reviewed (approved):
좋습니다!!!!
